app/screens/transactions/Transactions.tsx

```diff
@@ -187,17 +193,33 @@ const Transactions = ({ history }: RouteComponentProps) => {
     (state: RootState) =>
       state.wallet.accounts[state.wallet.currentAccountIndex].address
   );
+
+  const getNonce = useSelector(
```

Comment:
@ryukiller Hey,  basically getNonce, already has updated value, don't understand why we have use effect for it.